### ISSUE TITLE
Shuttle data

### ISSUE THREAD

**Issue by vonbearshark** [Pitt-CSC/club_project_2017#116]
Our Pitt API project has the shuttle API integrated. It would be an awesome addition to this

**Comment by SPN11** [comment]
What kinds of UI features / features in general are you looking for? There's a lot of functionality, and a TON of data haha. One could be a window containing estimates for each shuttles next stop location and time, or maybe filterable (Like uncheck a box, and the route disappears) highlighted routes for each shuttle. It'd be really tough to have live pins of each moving shuttle (and a lot of calls to the API), but those are just a couple thoughts.

**Comment by vonbearshark** [comment]
Yeah, I get the impression that live shuttle pins would bog us down, but having a pin per stop with an ETA for the shuttle would be helpful, I think

**Comment by SPN11** [comment]
Pin per stop could be a lot of pins, though. Perhaps a pin for the location of each shuttle's next stop? Then each popup would have the name of the shuttle, along with its ETA to that stop (These pins can be filtered by a checkbox)? Then underneath the legend (or have a "Shuttles" tab) maybe we could have a small table that contains this information? That way users don't have to click every pin to find the shuttle they're trying to track. Just some thoughts.

**Comment by vonbearshark** [comment]
Those are interesting ideas. Pick one and let's see what it looks like
